var/spack/repos/builtin/packages/py-neuroc/package.py

```diff
@@ -15,6 +15,7 @@ class PyNeuroc(PythonPackage):
     git = "ssh://bbpcode.epfl.ch/nse/NeuroC"
 
     version('develop', branch='master')
+    version('0.2.7', tag='neuroc-v0.2.7')
     version('0.2.6', tag='neuroc-v0.2.6')
```

Comment:
```suggestion

```

Reply:
What about 0.2.4? Could also always clean up later, 3 versions is quite a short history.

Reply:
there is only one other package depending on that, but yeah, let's add this new one for now

Reply:
should I also add 0.2.4 then?

Reply:
Ah nevermind, it's already there, sorry :innocent: